instagram/src/App.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import PostsPage from './components/PostContainer/PostsPage';
+import withAuthenticate from './authentication/withAuthenticate';
+import Login from './components/Login/Login';
+
+const ComponentFromWithAuthenticate = withAuthenticate(PostsPage)(Login);
+
+class App extends Component {
+  render() {
+    return(
+      <div className="App">
+        <ComponentFromWithAuthenticate />
```

Comment:
Great job cleaning this out, if it is not going to hold any state, I would recommend that you remember to go back and refactor your classes into functions to optimize performance.